pymc3/examples/rugby_analytics.py

```diff
@@ -0,0 +1,132 @@
+
+# coding: utf-8
+
+# # A Hierarchical model for Rugby prediction
+
+# # Based on http://danielweitzenfeld.github.io/passtheroc/blog/2014/10/28/bayes-premier-league/
+
+
+import numpy as np
+import pandas as pd
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+import pymc3 as pm3, theano.tensor as tt
```

Comment:
In the Ipython notebook this isn't like this. Do you want me to run the script and regenerate that python file?